Ewas proof read (#1447)
galaxyproject/training-material: topics/epigenetics/tutorials/ewas-suite/tutorial.md

# 450K Array Data Upload

> The first step of EWAS data anyalysis is the raw methylation data loading (intensity information files for each two color micro array)

Reviewer: ```suggestion
The first step of EWAS data analysis is the raw methylation data loading (intensity information files for each two color micro array)
```

> ### {% icon hands_on %} Hands-on: Preprocessing
> 1. Run **minfi mset** {% icon tool %} on the output from minfi_read450K to create `MethylSet` object
> 2. Run **minfi qc** {% icon tool %}  on the 'MethylSet' object created in the previous step to estimate sample-specific quality control. Once it is done click on the {% icon galaxy-eye %} (eye) icon next to the QC filename in your history to view QC image.
> 3.  Once you have inspected the quality control run **minfi rset** {% icon tool %} on the  'MethylSet'output from minfi_read450K  to convert methylation data from the `MethylSet` to a ratio of the methylated probe intensity and the overall intensity (sum of methylated and unmethylated probe intensities)
> 4. Run **minfi maptogenome** {% icon tool %} on the output from the previous step to map ratio data to the genomic positions
>
> > ### {% icon comment %} Preprocess and Normalize data

> > > If your files require normalisation, you might prefer to use on fo the other preprocessing tools provided in EWAS suite i.e. **minfi ppfun** {% icon tool %} or **minfi ppquantile**  {% icon tool %} look for recommendation at (ref).

Reviewer 2: this seems wrong?

Author: ```suggestion
> > If your files require normalisation, you might prefer to use one of the other preprocessing tools provided in EWAS suite i.e. **minfi ppfun** {% icon tool %} or **minfi ppquantile**  {% icon tool %} look for recommendation at (ref).
```
